### PR TITLE
Start ovn-northd on all the central nodes for cluster mode

### PR DESCRIPTION
**Pull request by numansiddique**
The one with the lock will do write txns to the SB DB.

Signed-off-by: Numan Siddique <numans@ovn.org>

### CODE REVIEW

File: ovn_cluster.sh [hunk 1]
```diff
@@ -368,6 +368,14 @@ function start-db-cluster() {
     --ovn-northd-nb-db=${REMOTE_PROT}:${central_1_ip}:6641,${REMOTE_PROT}:${central_2_ip}:6641,${REMOTE_PROT}:${central_3_ip}:6641 \
     --ovn-northd-sb-db=${REMOTE_PROT}:${central_1_ip}:6642,${REMOTE_PROT}:${central_2_ip}:6642,${REMOTE_PROT}:${central_3_ip}:6642 --ovn-manage-ovsdb=no \
     $SSL_ARGS start_northd
+    ${RUNC_CMD} exec ${CENTRAL_NAME}-2 ${OVNCTL_PATH}  \
```
Comment (flavio-fernandes) [inline ovn_cluster.sh:371]: Let's update the comment on line 366.
Should we make it more predictable on which one of the 3 northd will be
the one with the write lock? Maybe a config knob? If you do not mind,
please put a line in the commit that either points folks on knowing how
to control/show the state of the cluster.

File: ovn_cluster.sh [hunk 2]
```diff
@@ -40,7 +40,7 @@ IP_HOST=${IP_HOST:-170.168.0.0}
 IP_CIDR=${IP_CIDR:-16}
 IP_START=${IP_START:-170.168.0.2}
 
-OVN_DB_CLUSTER="${OVN_DB_CLUSTER:-no}"
+OVN_DB_CLUSTER="${OVN_DB_CLUSTER:-yes}"
```
Comment (flavio-fernandes) [inline ovn_cluster.sh:43]: on by default, huh? I'm kinda okay with it but we may soon need to think of
a set of defaults for simple/newcomers and one for exercising all the cool stuff
we have in ovn.

File: ovn_cluster.sh (hunk 1 above)
Comment (numansiddique) [reply ovn_cluster.sh:376]: Done.

File: ovn_cluster.sh (hunk 2 above)
Comment (numansiddique) [reply ovn_cluster.sh]: Oops. That was not the intention. I had modified it locally. I'd prefer cluster to be off by default.

**Pull request merged**